client/src/actions/ipfs.ts

```diff
@@ -135,6 +135,13 @@ export const saveFileToIPFS =
     }
 
     const fullContent: any = content;
+
+    const id = Number(currentProjectId);
+
+    if (currentProjectId && state.grantsMetadata[id]) {
```

Comment:
@schultztimothy if `currentProjectId` is `0` is not going to work. maybe we can check with something like

```
if (currentProjectId !== undefined && state.grantsMetadata[id] !== undefined)
```

Reply:
Good call!